autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -13,6 +13,7 @@ import { RootState } from '../../../../../../redux/reducer';
 import * as styles from './SectionSelect.css';
 import { updateCourseCard } from '../../../../../../redux/actions/courseCards';
 import GradeDist from './GradeDist/GradeDist';
+import sortMeeting from '../../../../../../utils/sortMeetingFunction';
```

Comment:
I think this would make more sense in the `Meeting.ts` folder. Thoughts @ryan-conn ?

Reply:
You mean the `Meeting.ts` file? Yeah I agree, I actually told him to put it in `utils` but since it relates specifically to meetings it makes sense to put them together